src/components/pages/home/hero/hero.jsx

```diff
@@ -82,7 +82,7 @@ const Hero = () => {
             {titleContentLg}
           </Heading>
           <p className="t-xl mt-7 max-w-[550px] text-white 2xl:mt-6 2xl:max-w-[480px] xl:mt-5 xl:max-w-[456px] lg:mx-auto lg:max-w-[414px]">
-            The fully managed multi-cloud Postgres with a generous free tier. We separated storage
+            The fully managed serverless Postgres with a generous free tier. We separated storage
```

Comment:
Consider changing to:

"We separate storage from compute to..."

Makes it sound a bit more dynamic and ongoing.

Reply:
Thank you. Fixed.